collector/README.md

```diff
@@ -1,39 +1,61 @@
 # Rust Compiler Performance Benchmarking
 
-Implements running benchmarks given a bors commit hash.
-
-## How to run
+## How to build
 
 ```
-# From repository root:
 cargo build -p collector --release
-./target/release/collector --benchmarks collector/benchmarks --output-repo $RUSTC_TIMING process
 ```
 
-$RUSTC_TIMING should point to a clone of the `https://github.com/rust-lang-nursery/rustc-timing`
-repository. Optionally, `--sync-git` can be passed to make the collector sync with the remote repo
-before and after comitting.
+## Benchmarking committed builds
+
+To benchmark builds from a rustc repository:
+```
+./target/release/collector --output-repo $RUSTC_TIMING process
+```
 
-### Running locally
+`$RUSTC_TIMING` should point to a clone of the `https://github.com/rust-lang-nursery/rustc-timing`
+repository, in which the output data will be placed and committed.
 
-Follow the standard instructions above, but run like this:
+## Benchmarking local builds
 
+To benchmark local builds:
 ```
-./target/release/collector --benchmarks collector/benchmarks --output-repo $RUSTC_TIMING \
-    bench_local $COMMIT_HASH $DATE $RUSTC
+./target/release/collector --output-repo $OUTPUR_DIR bench_local \
+    --rustc $RUSTC --cargo $CARGO $ID
 ```
 
-$RUSTC should point to the compiled rustc in stage1 or stage2, i.e.
-`./build/x86_64-unknown-linux-gnu/stage2/bin/rustc`. Stage 2 is likely a little better for
-benchmarking purposes, but not critical. The rustc binary has to be in a subdirectory of the
-`rustc-perf` root directory for the benchmark run to succeed.
+`$OUTPUT_DIR` should point to a directory, in which the timing data will be
+placed.
+
+`$RUSTC` should point to a rustc executable. For benchmarking purposes a stage
+2 build is a bit better than a stage 1 build, but not critical.
```

Comment:
We might want to mention that not all benchmarks will compile with a stage 1 build as they might need proc macros which won't work today at stage 1.

Reply:
Ok.